packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -113,11 +113,21 @@ function toTree(vnode) {
     }
     case HostText: // 6
       return node.memoizedProps;
+    case ForwardRef: {
+      return {
+        nodeType: 'function',
+        type: node.type,
+        props: { ...node.pendingProps },
```

Comment:
why pendingProps? is that all that's there on this fiber?

Reply:
Please don't use `pendingProps` anywhere.

In fact all of the fields Enzyme uses are subject to breakage. But I guess we talked about this before.

Reply:
Yeah I was really just feeling around in the dark here. I used pendingProps because that looked like it had the information I needed. Is it any worse than `memoizedProps` that the other tags use?

Reply:
`pendingProps` get cleared by React after rendering.